### ISSUE TITLE
RF no longer functions

### ISSUE THREAD

**Issue by Scrxtchy** [NightmareXIV/TextAdvance#27]
Since updating to v3.2.1.7, RF now throws this error
```
23:56:20.912 | ERR | [TextAdvance] The type initializer for 'ECommons.Automation.Callback' threw an exception.
	   at ECommons.Automation.Callback.FireRaw(AtkUnitBase* Base, Int32 valueCount, AtkValue* values, Byte updateState) in D:\VS\TextAdvance\ECommons\ECommons\Automation\Callback.cs:line 80
	   at ECommons.Automation.Callback.Fire(AtkUnitBase* Base, Boolean updateState, Object[] values) in D:\VS\TextAdvance\ECommons\ECommons\Automation\Callback.cs:line 137
	   at TextAdvance.Executors.ExecRequestFill.TryClickItem(AddonRequest* addon, Int32 i) in D:\VS\TextAdvance\TextAdvance\Executors\ExecRequestFill.cs:line 54
	   at ECommons.Automation.LegacyTaskManager.TaskManager.Tick(Object _) in D:\VS\TextAdvance\ECommons\ECommons\Automation\LegacyTaskManager\TaskManager.cs:line 140
```

You currently have to either disable the mode or press cancel to temporary disable it's functionality when fulfilling a request

**Comment by Limiana** [comment]
Hello!
I can not reproduce it.
Please send the entire log `%appdata%\XIVLauncher\dalamud.log` after function has failed

**Comment by Limiana** [comment]
Fixed in 3.2.1.8
